modules/configuration/ajax/process.php

```diff
@@ -29,7 +29,7 @@
 $client->initialize();
 
 $DB =& Database::singleton();
-foreach ($_POST as $key => $value) {
+foreach (array_unique($_POST) as $key => $value) {
     if (is_numeric($key)) { //update
```

Comment:
By duplication, you mean duplicate value? 
or duplicate key?

This patch will prevent multiple Config settings having the same value, should check for same key=value pair.  
You could also add a unique constraint in the DB on (ConfigID, Value). This way any attempt to add a duplication would generate a DB error

Reply:
@xlecours @ridz1208 @MounaSafiHarab @christinerogers Can any of you think of any reasons why the same config option would need to have the same value specified multiple times with the current LORIS config options?

Reply:
@PapillonMcGill 
Thanks for your comment.  Could u review again? I have added a unique key to the table.

Reply:
Nice to change TEXT to VARCHAR(768) but is the 768 value a little bit high? (maybe 100 or 150?)
With the UNIQUE key in DB, a DB error would be generated preventing multiple setting having the same value. 

But still using **array_unique**, that would keep only the first occurrence of a $value even if multiple  $key can have the same value (think YES/NO field)

Reply:
100 or 150 are not enough to save the data. The value needs to save "HTML code" and Document. But you are right, "array_unique" is a good choice in here.